lib/appsignal/cli/install.rb

```diff
@@ -227,10 +227,11 @@ def done_notice
 
         def installed_frameworks
           [].tap do |out|
-            out << :rails if framework_available? "rails"
             out << :sinatra if framework_available? "sinatra"
             out << :padrino if framework_available? "padrino"
             out << :grape if framework_available? "grape"
+            out << :rails if framework_available?("rails") &&
+              Appsignal::Utils::RailsHelper.application_config_exists?
```

Comment:
Looking at it better I think we should remove this check here and move it to the `install_for_rails` method. Then with a rescue block, also rescue the `LoadError` exception. It's not a problem that the file doesn't exist, but that when the file is loaded it references something which does not exist / is not loaded.